ms2/src/org/labkey/ms2/pipeline/mascot/MascotClientImpl.java

```diff
@@ -1523,6 +1525,11 @@ private InputStream getRequestResultStream (Properties parameters)
         @Test
         public void testMockMascotServer() throws IOException
         {
+            if (!AppProps.getInstance().isDevMode())
+            {
+                Assume.assumeNotNull("Mock mascot server requires DeveloperTools module. Skipping test in production mode.",

```

Comment:
Wouldn't it be better to just check for module presence and skip the production mode check? It's possible to have a devmode build that lacks this module.

Reply:
I've been following this pattern so that we don't start silently skipping tests when we shouldn't. Like, if that module's name changes without this getting updated.